### PR TITLE
Update slug, org, and project key info for the SonarCloud service

### PR DESCRIPTION
**Pull request by akosthekiss**
IoT.js-DCO-1.0-Signed-off-by: Akos Kiss akiss@inf.u-szeged.hu

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -99,7 +99,7 @@ matrix:
     - name: "SonarQube"
       addons:
         sonarcloud:
-          organization: "samsung-iotjs"
+          organization: "pando-project"
           token:
             secure: "u9HWNQNhAqQQdgl3yldKcQVH8plMQRwIdpzjsM4j3GBC4Wrh9u8guLJB3o003i0UsyaGg2knYFdLgOmEsgcvXAo2aIUyzf9CfK9RLRw5RtIuPMpmR7UjHdlf+QfCF+nY+BB2j0nAiWnxHve95du7sZflNxi+eNJJzquyBh1Wm8eqwoiRpCgiDzjRDEAUoz0FWMNny/x5545E970jpQ2bjHGx98tCMUO8ikINeL8sC99sumffaFONG8GVpwLjc8McfQfYpWbk0e0OPxZtGDyqKcyMxcbAGctklsigtsBZKlpj69uba3w4OSA3zJPCdQ4dKwCyBOcAAP8qeF5Jf0eLI8WLEgnKir2Pfc/rKkY0owuz7S+tUmizm3+T06wDFgwpLu0/PcA5oOcp4WpGXbAX7WujaAHB7YKAEsk324XC7Bdf+39OuZ0dbKWMiwU7rYV4NOYNPjN2BCb1XqyE0Ung41Ls6P4t/zwzYRZtiovhr6ibNBcwLVclfQZ/tbyBDuh++8dh7Ixe+x5RFiiCB0w/fiKqqXYM8/we4JU3f71y4DK6fP+nSN/vIYttvkN28HCCvBVSdyuuvPRM6Ro1yLNw9U9PHCJ1CIgcx8+I8Mep3PzBhDILXWjzlVu4sa/+aIoEq7MvWBDMhrFEP6RX+M6CiPmgj5+Lu/GZNivbu51RASI="
```
Comment (LaszloLango) [inline .travis.yml]: I think we need to update the secure token to make this work.

File: sonar-project.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-sonar.projectKey=samsung.iot.js
+sonar.projectKey=pando-project_iot.js
```
Comment (yichoi) [inline sonar-project.properties]: pando-project_iotjs

Comment (yichoi) [inline sonar-project.properties]: Could you please change it to ``pando-project_iotjs``?

File: README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 [![License](https://img.shields.io/badge/licence-Apache%202.0-brightgreen.svg?style=flat)](LICENSE)
 [![Build Status](https://travis-ci.org/pando-project/iotjs.svg?branch=master)](https://travis-ci.org/pando-project/iotjs)
 [![Coverity Scan Build Status](https://scan.coverity.com/projects/12140/badge.svg)](https://scan.coverity.com/projects/samsung-iotjs)
-[![SonarCloud Status](https://sonarcloud.io/api/project_badges/measure?project=samsung.iot.js&metric=alert_status)](https://sonarcloud.io/dashboard?id=samsung.iot.js)
+[![SonarCloud Status](https://sonarcloud.io/api/project_badges/measure?project=pando-project_iot.js&metric=alert_status)](https://sonarcloud.io/dashboard?id=pando-project_iot.js)
```
Comment (yichoi) [inline README.md]: Could you please change it to ``https://sonarcloud.io/dashboard?id=pando-project_iotjs``?

File: .travis.yml (hunk 1 above)
Comment (yichoi) [reply .travis.yml]: Now you can remove ``token``. I have generated token at sonarcloud then put it into travis directly by myself.

**Pull request merged**